mythos_testnet.yaml

```diff
@@ -0,0 +1,63 @@
+specVersion: 1.0.0
+name: subquery-nova-mythos
+version: 1.0.0
+runner:
+  node:
+    name: "@subql/node"
+    version: "*"
+    options:
+      historical: true
+  query:
+    name: "@subql/query"
+    version: "*"
+description: Nova SubQuery project
+repository: https://github.com/nova-wallet/subquery-nova
+schema:
+  file: ./schema.graphql
+network:
+  chainId: "0x15f6788bcf1d1a3b7e1c36074584e1a3f3d07e0a46e362a102c3c3df1a93669f"
+  endpoint: "wss://paseo-muse-rpc.polkadot.io"
+dataSources:
+  - name: main
+    kind: substrate/Runtime
+    startBlock: 2679576
```

Comment:
Is it first interesting block?

Reply:
Not the first one, but the one I had some rewards on. We can either start from the latest block on testnet or bisect the runtime versions to find one where collator staking was first introduced if we want the full history for testnet